.github/workflows/main.yml

```diff
@@ -92,11 +92,11 @@ jobs:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
Can you go through and comment on changes that are beyond the mechanical move and paths update that should be reviewed?